cloudigrade/util/models.py

```diff
@@ -0,0 +1,12 @@
+"""Cloudigrade Account Models."""
+from django.db import models
+
+
+class BaseModel(models.Model):
+    """Abstract model to add automatic created_at and updated_at fields."""
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
```

Comment:
Below here, we may want to add:

    ordering = ('created_at',)

so we have a default ordering for future models, but I don't feel too strongly about this right now.